tests/js/firstDiscoveryEditorTests.js

```diff
@@ -76,14 +76,18 @@ https://github.com/gpii/universal/LICENSE.txt
     };
 
     gpii.tests.firstDiscovery.hasClass = function (elementName, element, selector, hasClass) {
-        jqUnit.assertEquals("The visibility of " + elementName + " is " + (hasClass ? "visible" : "invisible"), hasClass, element.hasClass(selector));
+        jqUnit.assertEquals(elementName + " has the css class " + selector + " applied? " + (hasClass ? "yes" : "no"), hasClass, element.hasClass(selector));
```

Comment:
something about this seems strange, maybe just a rewording of the message would help simplify things.